docker-compose.yml

```diff
@@ -1,31 +1,13 @@
 version: "3.8"
 
 services:
-  mysql:
+  db:
     image: mysql:8.0.23
-    environment:
-      MYSQL_DATABASE: 'yourdbname'
-      MYSQL_USER: 'user'
-      MYSQL_PASSWORD: 'password'
-      MYSQL_ROOT_PASSWORD: 'rootpassword'
-    ports:
-      - "3306:3306"
-    volumes:
-      - mysql-data:/var/lib/mysql
     restart: always
-
-  app:
-    image: your-spring-boot-app:latest
-    depends_on:
-      - mysql
     environment:
-      SPRING_DATASOURCE_URL: jdbc:mysql://mysql:3306/yourdbname?useSSL=false&serverTimezone=UTC
-      SPRING_DATASOURCE_USERNAME: user
-      SPRING_DATASOURCE_PASSWORD: password
+      MYSQL_DATABASE: puresynth
+      MYSQL_USER: ${DB_USERNAME}
```

Comment:
It's ok to give simple credentials like `puresynth` in the compose file. This file is only for local and test development, so here are no special secrets. I don't know how well this work with env var.